src/generate-CST-derivings.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+############################################################################
+##  Copyright (C) 2017-2021 Yann Régis-Gianas, Nicolas Jeannerod,         ##
```

Comment:
Did you `shellcheck` this script shell?

Reply:
I did! (And I have ShellCheck on the fly in my editor.) The only thing it complains about is my use of `cat` line 109; and I do agree with ShellCheck that this `cat` is useless but I believe it improves clarity of the code.